docs/agents/sematext-agent/jmx-attaching-containers.md

```diff
@@ -0,0 +1,21 @@
+title: Automatic attaching to Java processes in container environments
+description: Sematext Agent can automatically attach to Java processes (Solr, Kafka, Zookeeper...) and expose JMX port required for monitoring from within the service itself
+
+When you assign a `MONITORING_TOKEN` to a Java service (e.g. via env variable), Sematext Agent will examine the process to see whether it can be
+monitored right away (i.e check if JMX ports are already exposed). If it can, Sematext Agent will start its monitoring. However, if it can't, Sematext Agent
+will try to inject into monitored process a special agent which has the sole purpose of opening a JMX port. JMX port opened in such way is
+protected by authentication based on temporary username and password known only to Sematext Agent.
+
+If JMX port is already open, but is protected with some authentication mechanism, Sematext Agent will try to open another JMX port anyway (it can't connect to the service
```

Comment:
If JMX port is already open, but is protected with some authentication mechanism, Sematext Agent will try to open another JMX port anyway (it can't connect to the service
=>
If a JMX port is already open but protected with authentication, Sematext Agent will try to open another JMX port anyway. It can't connect to the service